lms/djangoapps/teacher_dashboard/utils.py

```diff
@@ -0,0 +1,9 @@
+from django.conf import settings
```

Comment:
> I think it is redundant module.

We are taking from settings LABSTER_API_AUTH_TOKEN. I don't think that settings are reudndant